src/libs/auth.ts

```diff
@@ -118,8 +118,13 @@ const sendSignOutMetrics = async (cause: SignOutCause): Promise<void> => {
   });
 };
 
-export const signOut = (cause: SignOutCause = 'unspecified'): void => {
-  sendSignOutMetrics(cause);
+export const signOut = async (cause: SignOutCause = 'unspecified'): Promise<void> => {
+  try {
+    await sendSignOutMetrics(cause);
```

Comment:
Notes from our discussion... This is the key change. We don't actually need (or want) to wait for the metrics response. We only want to wait for the metrics request to be _sent_ before clearing the auth store. What we need to await here is the async `ensureAuthSettled` in `captureEvent`.